server/http_handler.go

```diff
@@ -334,6 +353,11 @@ type ddlHistoryJobHandler struct {
 	*tikvHandlerTool
 }
 
+// ddlResignOwnerHandler is the handler for resigning ddl owner.
+type ddlResignOwnerHandler struct {
+	*tikvHandlerTool
```

Comment:
This is not operated on TiKV.
How about we implement the method on `ddlResignOwnerHandler` directly?